src/App/Zinc.DeveloperCenter.Client.Web/projects/zn-developercenter/src/app/screens/adr-list/components/adr-summary/adr-summary-list.component.ts

```diff
@@ -32,10 +42,25 @@ export class AdrSummaryComponent implements OnDestroy {
     }
 
     public getAdrsForCurrentRepo(): void {
-        this.adrService.listAdrs(this.repoDotName)
+        this.adrService.listAdrs(this.repoDotName, this.sortedOn, this.sortAsc)
             .pipe(takeUntil(this.destroyed$))
             .subscribe(adrs => {
                 this.adrs = adrs;
+                this.updateLastUpdatedDates();
+            });
+    }
+
+    public updateLastUpdatedDates(): void {
+            this.adrs.items.forEach( (adr) => {
+                this.adrService.updateDates(this.repoDotName, adr.adrTitle)
+                    .subscribe(_lastUpdatedDate => {
+                        adr.lastUpdatedDate = _lastUpdatedDate;
+                        adr.lastUpdatedDateString = new Date(_lastUpdatedDate).toLocaleDateString(undefined, {
+                            year: 'numeric',
+                            month: 'long',
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 24 spaces but found 28.](https://app.codacy.com/gh/GSFSGroup/Zinc.DeveloperCenter/pullRequest?prid=9805944)
```suggestion
                        month: 'long',
```